src/app/modules/group/pages/managed-groups/managed-groups.component.ts

```diff
@@ -7,6 +7,9 @@ import { Component, OnInit } from '@angular/core';
 })
 export class ManagedGroupsComponent implements OnInit {
 
+  title = 'Groups you manage';
+  subtitle = 'Here are the groups you manage';
+
```

Comment:
Set/unset the title in breadcrumbs as well. Having "Groups you manage" as category is enough.

Reply:
Done, though I'm not sure if I did it the correct way.

Reply:
Do it in the init instead, put it to null when leaving the page (on destroy).

Reply:
Done